readme.md

```diff
@@ -164,104 +174,91 @@ to learn and work in any web development technologies.
 
 ### Weekly Schedule
 
-#### "Normal Weeks"
-
-- Mondays, Tuesdays, Wednesdays and Fridays
-  - 9:00-10:00:  Morning Exercise/Homework Review/Mini-lesson
-  - 10:00-12:30: Lesson 1 (Outcomes on Tuesdays)
-  - 12:30-2:30:  Lunch / Work Time / 1:1s / Office Hours
-  - 2:30-5:00:   Lesson 2
-  - 6:00-8:00:   Evening Office Hours (**Monday, Tuesday and Wednesday Nights**)
+#### Normal Weeks
 
-- Thursdays
-  - 9:00-10:00:  Morning Exercise/Homework Review/Mini-lesson
-  - 10:00-12:30: Lab w/ instructor support
-  - 12:30-2:30:  Lunch & Work Time
-  - 2:30-5:00:   Lab w/ instructor support
+Mondays, Tuesdays, Wednesdays and Fridays
+- **9:00-10:00**:  Code Challenge / Review / Mini-Lesson
+- **10:00-12:30**: Lesson 1 / Lab
+- **12:30-2:30**:  Lunch / Work Time / One-on-one's / Office Hours
+- **2:30-5:00**:   Lesson 2 / Lab / Outcomes (Wednesday)
+- **6:00-8:00**:   Evening Office Hours (Monday, Tuesday, Wednesday)
 
 #### Project Weeks (3, 7, 9, 12)
 
 Project weeks will be pretty much just time for you to work on your project.
-
 - There will be no scheduled lessons, except for a few optional mini-lessons.
 - There will be no evening office hours, since instructors will be widely available during the day.
-- In most cases, the Outcomes schedule will also remain the same during project weeks
+- The Outcomes schedule will remain the same during project weeks
 
 ### Squads and Rotations
 
 Each student will be part of a squad lead by one
-instructor. Your squad instructor will be the primary (but not the only)
+instructor. Your squad instructor will be the primary -- but not the only --
 instructor providing you with feedback on homework, labs, quizzes, etc.
 
-Though you'll stay together with your squad throughout the whole course, there
-will likely be at least one point in the course where instructors will rotate
-and lead a new squad.
-
 ### Instructor Communication
 
-- Additional Instructor Support is available during Evening Office Hours from
-6:00 - 8:00PM on Monday,Tuesday and Wednesday Nights. WDI instructors from other cohorts will be included in the Office Hours schedule.
-- Instructors who are not scheduled for Evening Office Hours **may not**
-respond to code questions via Slack or in person, even if we're in the office.
-  - **Question**: Why would we do this?
-  - **Answer**: Work/Life Balance.
-- Feel free to send any messages to us and we'll reach out to you the next day.
+<!-- AM: Could definitely use some revision here plz! -->
+
+There will be dedicated time for instructors to provide students with in-person support outside of lessons. These are limited to...
+* **Afternoon office hours**: Monday, Tuesday, Wednesday - 1:30PM - 2:30PM
+* **One-on-one's**: Monday, Tuesday, Wednesday - 1:30PM - 2:30PM
+* **Evening office hours**: Monday, Tuesday, Wednesday - 6:00PM - 8:00PM
+* **Lab support**
+
+Outside of these times, instructors **are not available** to provide in-person technical support, even if they are in the office.
```

Comment:
Will add that instructors are working on lessons, grading projects, etc.

I think we'll have emphasized the "solving your own problems" thing enough previously so will leave it to those sections.